jwala-services/src/main/java/com/cerner/jwala/service/webserver/component/WebServerStateSetterWorker.java

```diff
@@ -113,37 +122,18 @@ public void pingWebServer(final WebServer webServer) {
     }
 
     /**
-     * Checks if the web server is in webServerToPingSet, if it's not in the set it will be added to the set.
-     * This method is thread safe.
-     * @param webServer the web server
-     * @return true if the web server is in webServerToPingSet
-     */
-    private boolean isInWebServerToPingSet(final WebServer webServer) {
```

Comment:
Why remove this method and extract it to pingWebServer? It seems like we could still use it and make the pingWebServer method easier to read